src/Contract/Chain.purs

```diff
@@ -27,17 +25,17 @@ import Ctl.Internal.Types.Chain
 import Ctl.Internal.Types.Interval (POSIXTime)
 import Ctl.Internal.Types.Natural (Natural)
 
-getTip :: forall (r :: Row Type). Contract r Chain.Tip
-getTip = wrapContract QueryM.getChainTip
+getTip :: Contract Chain.Tip
+getTip = Contract.getChainTip
 
-waitUntilSlot :: forall (r :: Row Type). Slot -> Contract r Chain.Tip
-waitUntilSlot = wrapContract <<< QueryM.waitUntilSlot
+waitUntilSlot :: Slot -> Contract Chain.Tip
+waitUntilSlot = Contract.waitUntilSlot
```

Comment:
Can we just export the original function as we don't need to `wrap` them anymore?